examples/thoughts/.env.server

```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgresql://postgres:devpass1234@localhost:5432/thoughts-dev
```

Comment:
We may want to remove this file entirely since it is `.gitignore`'d and the only env file we have committed for examples.

Reply:
Yeah, thought so too. I can remove it.